File: IronVault/IronVault.Model/Models/Prisustvo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public partial class Prisustvo
    {
        public int PrisustvoId { get; set; }

        public int KorisnikId { get; set; }
        public Korisnik Korisnik { get; set; }

        public DateTime DatumVrijemeUlaska { get; set; }

        public DateTime? DatumVrijemeIzlaska { get; set; }

    }
}

File: IronVault/UI/ironvault_desktop/lib/models/prisustvo.dart
import 'package:ironvault_desktop/models/korisnik.dart';
import 'package:json_annotation/json_annotation.dart';

part 'prisustvo.g.dart';

@JsonSerializable()
class Prisustvo {
  int? prisustvoId;
  int? korisnikId;
  Korisnik? korisnik;
  DateTime? datumVrijemeUlaska;
  DateTime? datumVrijemeIzlaska;

  Prisustvo({this.prisustvoId});

  factory Prisustvo.fromJson(Map<String, dynamic> json) =>
      _$PrisustvoFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$PrisustvoToJson(this);
}
